pkg/project/project.go

```diff
@@ -0,0 +1,29 @@
+package project
+
+var (
+	description = "The ingress-exporter exports Prometheus metrics for tenant cluster ingress."
```

Comment:
Where does this run and how is it installed? It isn't installed by chart-operator in tenant clusters.

Reply:
It was introduced via https://github.com/giantswarm/giantswarm/issues/5199